### PR TITLE
Workflow Automated Test - LRQA-68856 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

**Comment by PauloVAS** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasks.path [hunk 1]
```diff
@@ -113,6 +113,26 @@
...
+<tr>
+	<td>ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_ICON</td>
+	<td>//button[contains(@class, 'btn btn-monospaced btn-unstyled')][2]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_RESULT_BAR</td>
+	<td>//span[contains(text(),'${key_resultBarMessage}')]</td>
+	<td></td>
+</tr>
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasks.path:130]: You can make the name of those paths more clear, you can use that same path to both areas (Assigned to me / Assigned to my roles)
Suggestion:

MY_WORKFLOW_TASKS_SEARCH_INPUT_FIELD
MY_WORKFLOW_TASKS_SEARCH_ICON
MY_WORKFLOW_TASKS_SEARCH_RESULT_BAR

Remember to change the names at the created macro (searchAssignedToMe)

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasks.path [hunk 2]
```diff
@@ -113,6 +113,26 @@
...
+<tr>
+	<td>ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_FINDS_NO_RESULTS</td>
+	<td>//div[contains(@class,'sheet-text text-center')]</td>
+	<td></td>
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasks.path:134]: There is no need to create this path.

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 3]
```diff
@@ -270,12 +270,39 @@ definition {
...
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+		Workflow.searchAssignedToMyRoles(searchTerm = "@");
+
+		AssertTextEquals(
+			locator1 = "MyWorkflowTasks#ASSIGNED_TO_MY_ROLES_TABLE_SEARCH_FINDS_NO_RESULTS",
+			value1 = "There are no tasks assigned to your roles with the specified search criteria.");
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:298]: Here you can use :
```
AssertTextEquals(
			locator1 = "Message#EMPTY_INFO",
			value1 = "There are no tasks assigned to you with the specified search criteria.");
```

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Comment by PauloVAS** [comment]
ci:test:sf

**Comment by PauloVAS** [comment]
@lucasperj, fixed!

**Review by lucasperj — APPROVED** [review]

**Comment by lucasperj** [comment]
Just started reviewing :)

**Comment by lucasperj** [comment]
Combined and sent on [PR#3](https://github.com/lucasperj/liferay-portal/pull/3). Thank you!

**Pull request closed without merging**